meta/change_management.md

```diff
@@ -0,0 +1,65 @@
+
+Note that changes to this document are also governed by the process described in this document.
+
+## Phase 1: Version 1 Development
+
+The initial version development of this project consists of all work conducted on or after Sunday, November 1, 2020 through to the beginning of phase 2, as defined by phase 2. There are 4 developers involved in phase 1:
+
+- Gryphon Shafer
+- Scott Peterson
+- Jeremy Swingle
+- Zachary Tinker
+
+During the initial version development of this project, all changes will be made as follows:
+
+1. A change's author should submit the change as a *Pull Request* (PR) to the project's `master` branch
+2. At least 1 developer other than the author should comment on the PR; however, ideally many developers will comment
+3. Discussion, debate, and revisions to the PR happen all within GitHub's PR system
+4. The PR is merged to `master` when any of the following is true:
+    - 1 developer other than the author agrees with the change and there are no disagreements from the other developers after 1 week from PR submission
+    - 2 developers other than the author agree with the change
+
+## Phase 2: Version 1 Quality Assurance
+
+Phase 2 begins once all 4 developers agree that all primary writing and editing as part of phase 1 has concluded. Additional editing may still take place, but at this point, all 4 developers believe they have a working product that, while potentially containing errors, is not likely missing information.
+
+The following are the steps of phase 2:
+
+1. Gryphon conducts an "OCD pass" of all content, making it align exactly with syntax, style, and other requirements
+2. Gryphon submits a PR for review by the other 3 developers
+3. Discussion and debate take place as part phase 1 PRs, and eventually the PR is merged to `master`
+4. Gryphon builds a single *Adobe PDF file* (PDF) for the entire project
+5. Gryphon creates a PR covering the entire project
+
+## Phase 3: CQLT Review and Approval
+
+The PDF will be submitted to the CQLT for review and approval. The 6 quizzing members of the CQLT will have 1 calendar month from the date of submission to review and provide feedback on the submission. Any feedback other than general approval should be made as GitHub comments to the PR covering the entire project.
+
+The 4 developers will respond to any feedback, potentially creating subsequent versions of the output from phase 2 as necessary.
+
+At the end of the 1 month review period of phase 3, if there are no unresolved comments regarding *functional equivalence* (FE) variance, the 6 quizzing members of the CQLT will vote to approve.
+
+## Phase 4: District Ratification
+
+Following the conclusion of phase 3, Zachary will take a resulting final PDF of the whole of the project's content and submit this to all 16 active districts. Gryphon will make available a ratification PR similar to the PR from phase 3.
```

Comment:
Is there a maintained and publicly available list of these "active" districts?

Reply:
Yes. 

https://www.cmabiblequizzing.org/districts.html & On the resouces page under "Documents from CQLT" https://docs.google.com/spreadsheets/d/1RIan6Wk5hiH6RonGsVY2lAI1FS3NMSio1SmeXx3J5vU/edit?usp=sharing

There is also a form on the district page that allows for districts to update their contact information that way, or by replying to our yearly call of district updates to be sent to us in September.

Reply:
Do we have a definition of "active?" Asking because Gryphon used it. And because there are 17 districts in your list Zach, and Gryphon referenced 16 (thinking MidAmerican isn't "active yet" but that's just me guessing).

Reply:
I just added Mid-American this week as Heather is working to coordinate a new group both now and over the past few months. They were not active, however they are now becoming active. At this point the district would not be considered as a true district as they do not have a district approved district coordinator, nor are they actively meeting as they are still in their infancy.

Reply:
The specifics of an "active" and "true" quizzing district probably don't matter that much--we want all districts to be able to vote, and no one will take the effort to create a sham district just to get an extra vote. 

BUT, pretty sure it's been a loooooong time since PNW's district coordinator has been approved by anyone in the actual CMA district.

Reply:
That was a bit of a misspeak on my part. District offices only have to approve of CQLT nominations and of quizzing in their district according to the rulebook & bylaws

Each **district** shall determine eligibility guidelines for all local, zone, and district competitions subject to approval by the **district office**. (This gives the impression that district offices have to approve of quizzing since they have to approve any eligibility guidelines)

Nominees must have the approval of the District quizzing point person, the **District Superintendent** or his appointee and the nominee’s pastor.

Reply:
I can pull some records and gather every time a district has participated in communication.... I will look for: replied emails, sent (from the district) emails, fee payment, IBQ participation, and CQLT election votes.

Reply:
I don't really care what it is, and I bet Zach could come up with a good definition that isn't overly onerous or exclusionary.